Tasks/Inc/BPS_Tasks.h

```diff
@@ -1,32 +1,60 @@
 #ifndef TASK_H__
 #define TASK_H__
 /* Kernel includes. */
-#include "FreeRTOS.h" /* Must come first. */
-#include "task.h" /* RTOS task related API prototypes. */
-#include "queue.h" /* RTOS queue related API prototypes. */
-#include "timers.h" /* Software timer related API prototypes. */
-#include "semphr.h" /* Semaphore related API prototypes. */
+#include "FreeRTOS.h"       /* Must come first. */
+#include "task.h"           /* RTOS task related API prototypes. */
+#include "queue.h"          /* RTOS queue related API prototypes. */
+#include "timers.h"         /* Software timer related API prototypes. */
+#include "semphr.h"         /* Semaphore related API prototypes. */
+#include <event_groups.h>   /* Event groups */
 
+#include "stm32xx_hal.h"
```

Comment:
Including this will include the FreeRTOS headers as well, so you don't need to doubly include them.